SUMMARY.md

```diff
@@ -33,10 +34,12 @@
 * [Replica Operations](reference/replica-operations.md)
 * [Call-home metrics](reference/call-home.md)
 * [Tested Third Party Software](reference/tested-third-party-software.md)
+* [ETCD Migration](quickstart/etcdmigration.md)
+* [Scale ETCD Members](quickstart/scaling_etcd.md)
```

Comment:
Need consistency I feel:
- The headers can be "Migrate ETCD" and "Scale ETCD"
- The md file names should be migrate-etcd.md and scaling-etc.md
Please note the "-" instead of "_" as we seem to be following that norm.

Reply:
Agreed !